### PR TITLE
Refactor constant scoping

### PR DESCRIPTION
**Pull request by markusewalker**
### Issue: https://github.com/rancher/qa-tasks/issues/1285

Speaking further offline with the team, we agreed that it would be best to refactor some of the scoping locally to packages where it makes sense. Before, everything was being global scoped, which is not a practice we should be pursuing long term.

### CODE REVIEW

**Review by slickwarren — COMMENTED** [review]
initial review lgtm; pending jenkins run for approval

**Review by Josh-Diamond — APPROVED** [review]

**Review by slickwarren — APPROVED** [review]

**Pull request merged**
